<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>чФицИ╖ц│ихЖМ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>чФицИ╖хРН</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>хпЖчаБ</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmPasswordEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>чбошодхпЖчаБ</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="togglePasswordButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ЁЯСБя╕П</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleConfirmPasswordButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ЁЯСБя╕П</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerConfirmButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>чбошодц│ихЖМ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToLoginButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ш┐ФхЫЮчЩ╗х╜Х</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>